UI/src/view/point_asset/trafficSignLabel.js

```diff
@@ -137,6 +237,11 @@
       return this.value && (this.value > 0 && this.value < 100000);
     };
 
+    var validateAdditionalInfo = function () {
+      var labelMaxLength = getLabelProperty(this).getMaxLength();
```

Comment:
There is any reason to return` this.value` if the method is for AdditionalInfo validation ?